papis/defaults.py

```diff
@@ -54,6 +54,7 @@ def get_default_opener() -> str:
     "doctor-duplicated-keys-keys": ["ref"],
     "doctor-html-codes-keys": ["title", "author", "abstract", "journal"],
     "doctor-html-tags-keys": ["title", "author", "abstract", "journal"],
+    "doctor-key-type-check-separator": " ",
```

Comment:
Note to self: not sure what a good default would be here? Probably just `None`..

Reply:
And yes, I think `None` would be good. Better not to assume anything here...

Reply:
Changed in the latest version. Thanks!